<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestionCategorias</class>
 <widget class="QMainWindow" name="GestionCategorias">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestión de Categorías</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="nombreLabel">
        <property name="text"><string>Nombre:</string></property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="nombreLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="descripcionLabel">
        <property name="text"><string>Descripción:</string></property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTextEdit" name="descripcionTextEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonsLayout">
      <item>
       <widget class="QPushButton" name="crearCategoriaPushButton">
        <property name="text"><string>Crear</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="listarCategoriasPushButton">
        <property name="text"><string>Listar Categorías</string></property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="categoriasTableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="editarEliminarLayout">
      <item>
       <widget class="QPushButton" name="editarCategoriaPushButton">
        <property name="text"><string>Editar Categoría</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="eliminarCategoriaPushButton">
        <property name="text"><string>Eliminar Categoría</string></property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>